server.js

```diff
@@ -32,9 +32,30 @@ const { createState } = require("./utils/state")
 // set the view engine to ejs
 app.set('view engine', 'ejs')
 
+app.use('/public/', express.static('./public'));
+
+// Load theme data
```

Comment:
**suggestion:** See the PR comment below about not wanting to encourage folks to dynamically update the theme data at runtime.

We could still encourage folks in this example code to use a code structure that is configurable...but from a _static_ configuration which they've configured offline.

So the comment and corresponding code could be something like: _"Load theme data configured for this financial institution. For performance reasons, and the fact that financial institutions typically do not update their themes very often, it's best to fetch the theme data offline and store the configuration for use in the live plugin."_

Reply:
I've removed the theme retrieval logic.